package.json

```diff
@@ -34,6 +34,7 @@
     "onLanguage:rmd",
     "onLanguage:debian-control.r",
     "workspaceContains:*.{rproj,Rproj,r,R,rd,Rd,rmd,Rmd}",
+    "workspaceContains:*/*.{rproj,Rproj,r,R,rd,Rd,rmd,Rmd}",
```

Comment:
```suggestion
    "workspaceContains:**/*.{rproj,Rproj,r,R,rd,Rd,rmd,Rmd}",
```

Reply:
`**/*.{}` means recursive matching. For large and deep workspace folder, it might slow down the startup.

Reply:
OK. I think it's enough to match one subfolder layer only.